### PR TITLE
Rename and update docs (metricstory -> convertly)

### PR DESCRIPTION
**Pull request by tcgilbert**
### Proposed changes

Rename and update docs (metricstory -> convertly)

### Merge timing
asap

### CODE REVIEW

File: src/connections/destinations/catalog/convertly/index.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+5. Copy your API key.
+6. Enter the API key in the destination settings in Segment.
+
+## Supported methods
+
+Convertly supports the following methods, as specified in the [Segment Spec](/docs/connections/spec).
+
+### Page
+
+Send Page calls to ADD WHAT PAGE CALLS ARE USED FOR HERE. For example:
```
Comment (tcgilbert) [inline src/connections/destinations/catalog/convertly/index.md:26]:
```suggestion
Send Page calls to ADD WHAT PAGE CALLS ARE USED FOR HERE. For example:
```

@randallkanna can you advise what should go here in place of "ADD WHAT PAGE CALLS ARE USED FOR HERE"

File: src/connections/destinations/catalog/convertly/index.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+```js
+analytics.page()
+Segment sends Page calls to Convertly as a pageview.
+```
+
+### Identify
+
+Send Identify calls to ADD WHAT IDENTIFY CALLS ARE USED FOR HERE. For example:
```
Comment (tcgilbert) [inline src/connections/destinations/catalog/convertly/index.md:35]:
```suggestion
Send Identify calls to ADD WHAT IDENTIFY CALLS ARE USED FOR HERE. For example:
```

@randallkanna can you advise what should go here in place of "ADD WHAT IDENTIFY CALLS ARE USED FOR HERE"

File: src/connections/destinations/catalog/convertly/index.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+analytics.identify("userId123", {
+  email: "john.doe@example.com"
+});
+```
+
+Segment sends Identify calls to Convertly as an identify event.
+
+### Track
+
+Send Track calls to ADD WHAT Track CALLS ARE USED FOR HERE. For example:
```
Comment (tcgilbert) [inline src/connections/destinations/catalog/convertly/index.md:47]:
```suggestion
Send Track calls to ADD WHAT Track CALLS ARE USED FOR HERE. For example:
```

@randallkanna can you advise what should go here in place of "ADD WHAT TRACK CALLS ARE USED FOR HERE"

File: src/connections/destinations/catalog/convertly/index.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
+---
+title: Convertly Destination
+id: 65e8b496eec9c40dbccbf749
+beta: true
+---
+
+[Convertly](https://www.tryconvertly.com){:target="\_blank”} lets you run AI on your product analytics. Create and generate charts and analyze data in minutes.
+
+This destination is maintained by Convertly. For any issues with the destination, contact the [Convertly support team](support@tryconvertly.com).
```
Comment (forstisabella) [inline src/connections/destinations/catalog/convertly/index.md:9]:
```suggestion
This destination is maintained by Convertly. For any issues with the destination, contact the [Convertly support team](mailto:support@tryconvertly.com).
```

File: src/connections/destinations/catalog/convertly/index.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+## Getting started
+
+1. From the Destination catalog page in the Segment app, search for Convertly.
+2. Select and click Add Destination.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/convertly/index.md:14]:
```suggestion
2. Select and click **Add Destination**.
```

File: src/connections/destinations/catalog/convertly/index.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+3. Select an existing Source to connect to.
+4. In Convertly, navigate to your [API Keys](https://www.app.tryconvertly.com/account/apikeys){:target="\_blank"} page in the Convertly app.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/convertly/index.md:16]:
```suggestion
4. In Convertly, navigate to your [API Keys](https://www.app.tryconvertly.com/account/apikeys){:target="\_blank"} page.
```

File: src/connections/destinations/catalog/convertly/index.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+5. Copy your API key.
+6. Enter the API key in the destination settings in Segment.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/convertly/index.md:18]:
```suggestion
6. Return to Segment and enter the API key in the destination settings for your Convertly destination.
```

**Review by forstisabella — CHANGES_REQUESTED** [review]

File: src/connections/destinations/catalog/convertly/index.md (hunk 3 above)
Comment (randallkanna) [reply src/connections/destinations/catalog/convertly/index.md:48]: Yes! Thanks Thomas!

Track calls are used with event names to send to Convertly to track user events. Once a track event is sent, Convertly uses it to query with natural language.

File: src/connections/destinations/catalog/convertly/index.md (hunk 2 above)
Comment (randallkanna) [reply src/connections/destinations/catalog/convertly/index.md:36]: Identify calls are used in Convertly to identify users with a unique ID that Convertly can use to build charts and tables to see how your users are interacting with data. All in natural language.

File: src/connections/destinations/catalog/convertly/index.md (hunk 1 above)
Comment (tcgilbert) [reply src/connections/destinations/catalog/convertly/index.md:26]: @randallkanna thanks! and this one here^

**Review by forstisabella — DISMISSED** [review]
once the fillers ("ADD WHAT Track CALLS ARE USED FOR HERE") are removed we'll be g2g!

**Comment by tcgilbert** [comment]
> once the fillers ("ADD WHAT Track CALLS ARE USED FOR HERE") are removed we'll be g2g!

I think they are there ([in this commit](https://github.com/segmentio/segment-docs/pull/6809/commits/ecd268683630a7011b2293f9dbe3b9b4194de75e)), idk why the file tab is not showing the changes though?

**Comment by tcgilbert** [comment]
> once the fillers ("ADD WHAT Track CALLS ARE USED FOR HERE") are removed we'll be g2g!

apologies missed that, its fixed now

**Review by forstisabella — APPROVED** [review]

**Comment by forstisabella** [comment]
> > once the fillers ("ADD WHAT Track CALLS ARE USED FOR HERE") are removed we'll be g2g!
> 
> apologies missed that, its fixed now

LGTM!

**Pull request merged**